packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/TargetAddressLabel/index.tsx

```diff
@@ -19,6 +19,7 @@ const TargetAddressLabel = ({ target, type }: Props) => {
         <Translation id="TR_SENT_TO_SELF" />
     ) : (
         target.addresses?.map((a, i) =>
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
Doesn't address have some id that could be used here?

Reply:
not really, there is only address itself. I assume that tx can have multiple outputs going to the same address. I am correct @szymonlesisz?

Anyway using index is fine in this case as the array is always the same (we are not changing its items or reordering it)

Reply:
yes its safe :) comment why